src/pages/128x64x1/timer_page.c

```diff
@@ -59,21 +59,21 @@ static int row_cb(int absrow, int relrow, int y, void *data)
     y += space;
     /*prem-timer reset */
     GUI_CreateLabelBox(&gui->resetpermlbl, LABEL_X, y,
-            LABEL_WIDTH, LINE_HEIGHT, &LABEL_FONT, NULL, NULL, _tr("Reset"));
+            LABEL_WIDTH, LINE_HEIGHT, &LABEL_FONT, GUI_Localize, NULL, _tr_noop("Reset"));
     GUI_CreateButtonPlateText(&gui->resetperm, RESET_X, y,
             RESET_WIDTH, LINE_HEIGHT, &BUTTON_FONT, show_timerperm_cb, reset_timerperm_cb,(void *)(long)absrow);
     if(Model.mixer_mode != MIXER_STANDARD) {
         /* or Reset switch */
     	GUI_CreateLabelBox(&gui->resetlbl, LABEL_X, y,
-            LABEL_WIDTH, LINE_HEIGHT, &LABEL_FONT, NULL, NULL, _tr("Reset sw"));
+            LABEL_WIDTH, LINE_HEIGHT, &LABEL_FONT, GUI_Localize, NULL, _tr_noop("Reset sw"));
     	GUI_CreateTextSourcePlate(&gui->resetsrc, TEXTSEL_X, y ,
             TEXTSEL_WIDTH, LINE_HEIGHT, &TEXTSEL_FONT, toggle_resetsrc_cb, set_resetsrc_cb, set_input_rstsrc_cb, (void *)(long)absrow);
 	y += space;
```

Comment:
Lint Error:
```
70:  Use int16/int64/etc, rather than the C type long  
71:  Tab found; better to use spaces  
73:  Should have a space between // and comment  
75:  At least two spaces is best between code and comments  
```